components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -145,10 +148,10 @@ void BraveReferralsService::Start() {
 
   // On first run, read the promo code from user-data-dir and
```

Comment:
I think this comment is no longer correct, as this no longer occurs "just" on first run.